src/boundingboxes.jl

```diff
@@ -29,6 +29,7 @@ boundingbox(p::Point) = Box(p, p)
 
 boundingbox(b::Box) = b
 
+# TODO: should the coordinate function return a Vec?
```

Comment:
the coordinates function should probably return the coordinate type now. we need to refactor the code to make sure that callers assuming Vec will instead call `point - origin`